<?xml version="1.0"?>
<launch>
    <!-- <include file="$(find img_input)/launch/img_input.launch" /> -->
    <include file="$(find v10_pillar_detector)/launch/v10_pillar_detector.launch" />

    <node pkg="v12_goalpost_detector" type="v12_goalpost_detector_node" name="v12_goalpost_detector_node" output="screen" >
<!--             <param name="variance" type="double" value="2.0" />    -->
        <remap from="/v12_goalpost_detector_node/image_in" to="/img_input_node/image_out" />
        <remap from="/v12_goalpost_detector_node/camera_info_in" to="/usb_cam/camera_info" />
        <remap from="/v12_goalpost_detector_node/frame_mode" to="/v12_goalpost_monitor_node/frame_mode" />
        <remap from="/v12_goalpost_detector_node/LUT_data" to="/v12_goalpost_monitor_node/LUT_data" />
        <remap from="/v12_goalpost_detector_node/save_param" to="/v12_goalpost_monitor_node/save_param" />
        <remap from="/v12_goalpost_detector_node/goalpost_ref" to="/v12_goalpost_monitor_node/goalpost_ref" />
        <remap from="/v12_goalpost_detector_node/line_tip" to="/v9_localization_node/line_tip" />
        <param name="ball_config_path" type="string" value="$(find v12_goalpost_detector)/config/saved_config.yaml" />
    </node>
</launch>
